mentoring/mentoring.py

```diff
@@ -235,6 +279,44 @@ def next_step_url(self):
         return '/jump_to_id/{}'.format(self.next_step)
 
     @XBlock.json_handler
+    def get_results(self, queries, suffix=''):
+        results = []
+        if not self.show_extended_feedback():
+            return {
+                'results': [],
+                'error': 'Results cannot be obtained.'
+            }
+        completed = True
+        choices = dict(self.student_results)
+        step = self.step
+        # Only one child should ever be of concern with this method.
+        for child in self.get_children_objects():
+            if child.name and child.name in queries:
+                results = [child.name, child.get_results(choices[child.name])]
+                completed = choices[child.name]['status']
+                break
+
+        message = self.get_message(True)
```

Comment:
@e-kolpakov Nevermind, looks like it's more complicated than that. It appears to be a way to appease oddly named and conditioned variables down the line in the JavaScript that existed beforehand. I'll see if there's some simple way to refactor this and make things clearer.